rust/README.md

```diff
@@ -70,15 +70,11 @@ Alternatively, you can replace `cargo run` with `bazel run //rust:selenium-manag
-| M8: Auto-installable Selenium Manager in Linux            | <ul><li>Implement the auto installation feature for Linux.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                               |
-| M9: Cross-platform auto-installable Selenium Manager      | <ul><li>Implement the auto installation feature for all the operating systems.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                           |
-
+| **Milestone**                           | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| M1: Driver management                   | <ul><li>Beta version of the Selenium Manager.</li> <li>Focused on driver management for Chrome, Firefox, and Edge.</li> <li>Selenium Manager compiled for Windows, Linux, and macOS (in GH Actions).</li> <li>Available in Selenium binding languages (Java, JavaScript, Python, Ruby, and C#).</li> <li>Used as a fallback for language bindings, when the driver is not found.</li> <li>Selenium Manager binaries bundled within the binding languages.</li></ul> |
```

Comment:
msedgedriver isn't working right now; I've set it to ignore in the bindings, I say we push it off to M2, but depends on how hard is the fix.

Reply:
This is the first notice I have about that. What is the problem? In any case, I think this PR is not the best place to report a issue like that. If I know the problem, I can try to solve it for M1.